pkg/openapi/openapi_generated.go

```diff
@@ -9834,7 +9798,7 @@ func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenA
 					Properties: map[string]spec.Schema{
 						"replicas": {
 							SchemaProps: spec.SchemaProps{
-								Description: "Replicas is the number of desired replicas. This is a pointer to distinguish between explicit zero and unspecified. Defaults to 0. More info: https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller#what-is-a-replicationcontroller",
+								Description: "Replicas is the number of desired replicas. This is a pointer to distinguish between explicit zero and unspecified. Defaults to 1. More info: https://kubernetes.io/docs/concepts/workloads/controllers/replicationcontroller#what-is-a-replicationcontroller",
```

Comment:
This shouldn't change with this PR. Is it a rebasing issue?

Reply:
This got in as `hack/generatecode` was not run previously. Though we need to check on the open-api generate-code path, as in reality it gets defaulted to `0`, but not the concern of this PR.